ui/src/demos/named-entity-recognition/Main.tsx

```diff
@@ -41,18 +40,12 @@ export const Main = () => {
                             }>
                             {({ input, model, output }) => (
                                 <Output>
-                                    <Output.Section
+                                    <Output.ShareSection
```

Comment:
```suggestion
                                    <Output.ShareableSection
```

Though I'm not a big fan of this generalization. This is a place where I think it's cleaner to keep them separate.

The rationale is that their combination feels a little awkward. Sure, they're often used together, but combining them feels like we're forcing two things into a bucket together that aren't terribly related.  Maybe it's ok and preferable because it saves a few lines, but my gut instinct doesn't like it.

Reply:
lets try it, this will be in every page, and users of main really dont care about how share works

Reply:
Sure, maybe we update the name though?